pkgs/development/python-modules/keymap-drawer/default.nix

```diff
@@ -18,18 +18,18 @@
   versionCheckHook,
 }:
 let
-  version = "0.22.1";
+  buildPythonPackage' = fn: lib.fix (self: buildPythonPackage (fn self));
 in
-buildPythonPackage {
-  inherit version;
+buildPythonPackage' (self: {
```

Comment:
Nitpick: Since throughout Nixpkgs we use the `finalAttrs` name here, I'd recommend to use it here too.